### PR TITLE
added kubernetes code sample

### PR DESCRIPTION
**Pull request by eminn**

### CODE REVIEW

File: co-group/src/main/java/BatchCoGroup.java [hunk 1]
```diff
@@ -123,10 +124,18 @@ private static Pipeline coGroupBuild() {
         return p;
     }
 
+    private static JetInstance getJetInstance() {
```
Comment (cangencer) [inline co-group/src/main/java/BatchCoGroup.java]: this also needs some explanation

File: integration/kubernetes/README.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+Starting a Hazelcast Jet cluster consists of a few steps: 
+
+- Creating Role Binding
+- Creating Config Map
+- Starting Hazelcast Jet Cluster
+
+
+### Creating Role Binding
+
+Hazelcast Jet uses Kubernetes API to discover nodes and that is why you need to grant certain permissions. The simplest Role Binding file can look as `rbac.yaml`. Note that you can make it more specific, since Hazelcast actually uses only certain API endpoints. Note also that if you use "DNS Lookup Discovery" instead of "REST API Discovery", then you can skip the Role Binding step at all. Read more at [Hazelcast Kubernetes API Plugin](https://github.com/hazelcast/hazelcast-kubernetes).
```
Comment (cangencer) [inline integration/kubernetes/README.md]: missing new lines. Text files should be wrapped at 72 chars. Goes for whole file.

File: integration/kubernetes/README.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+```
+
+## Running a Job on Hazelcast Jet Cluster
+
+There are two different ways to submit a job to a Hazelcast Jet cluster.
+
+1. Package Job as a Docker container then let it submit itself. 
+2. Submit Job as a JAR file from a shared `PersistentVolume` which is attached to a pod.
+
+We will use [Co-Group Transform](https://github.com/hazelcast/hazelcast-jet-code-samples/blob/0.6-maintenance/batch/co-group/src/main/java/CoGroup.java) code sample from [Hazelcast Jet Code Samples](https://github.com/hazelcast/hazelcast-jet-code-samples) repository to submit it as a job.
```
Comment (cangencer) [inline integration/kubernetes/README.md]: points to 0.6 version

File: integration/kubernetes/README.md [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+$ git clone git@github.com:hazelcast/hazelcast-jet-code-samples.git
+```
+
+Navigate to `co-group/` folder
+
+```bash
+$ cd co-group/
+```
+
+The code sample already has a `jib` profile which enables jib build configured. You might need to add your docker registry credentials to there or if you'd like to use your local registry, please update [lifecycle binding](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#bind-to-a-lifecycle) goal to `dockerBuild`  from `build` in the `pom.xml`.
```
Comment (cangencer) [inline integration/kubernetes/README.md]: some explanation of what jib does would be good

File: integration/kubernetes/README.md [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+
+![](images/Dashboard.png)
+
+
+
+## Inspecting the Job with Hazelcast Jet Management Center
+
+Hazelcast Jet Management Center enables you to monitor and manage your cluster members running Hazelcast Jet. In addition to monitoring the overall state of your clusters, you can also analyze and browse your jobs in detail.
+
+You can check [Hazelcast Jet Documentation](http://docs.hazelcast.org/docs/jet/latest-dev/manual) and [Hazelcast Jet Management Center Documentation]() for more information.
```
Comment (cangencer) [inline integration/kubernetes/README.md]: missing link

**Comment by cangencer** [comment]
Also missing mention in main readme

File: integration/kubernetes/README.md [hunk 6]
```diff
@@ -0,0 +1,341 @@
...
+$ cd co-group/
+```
+
+Then, build the co-group code sample with the command below: 
+
+```bash
+$ mvn clean package
+```
+
+It should output the file `co-group-0.7-jar-with-dependencies.jar` under the
```
Comment (gurbuzali) [inline integration/kubernetes/README.md]: It would be good if we can make this `version-agnostic`

**Pull request merged**